src/main.cpp

```diff
@@ -85,6 +96,13 @@ int main(int argc, char *argv[])
         }
     }
 
+    auto end_time = std::chrono::high_resolution_clock::now();
+    auto duration_ms = std::chrono::duration_cast<std::chrono::milliseconds>(end_time - start_time);
+    auto duration_s = std::chrono::duration_cast<std::chrono::seconds>(end_time - start_time);
+
+    std::cout << "Render time: " << duration_ms.count() << "ms" << std::endl;
+    std::cout << "Render time: " << duration_s.count() << "s" << std::endl;
```

Comment:
Wanted to show the rendertimes in milliseconds and in seconds (two different resolutions).

We can also define a `TimeElapsed` class or something like this, that shows for us the correctly printed render time, in the format: `HH:MM:SS:ms`.